src/style.css

```diff
@@ -0,0 +1,22 @@
+@import "./blocks/card.css";
+@import "./blocks/stars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+h3,
```

Comment:
maybe here it is better to use classes? According to the checklist p1 => `Get used to style all elements using classes. And don't increase selectors specificity unless completely necessary.`

Reply:
but in lectures Misha did this in reset file:
body, h1, h2, h3, ul, p {
  margin: 0;
}

Reply:
@vinnykalisa So it is better to ask the mentor as a checklist seems a little controversial with Misha's lectures. Maybe it is an exeption

Reply:
If you'll need it later - they told be that for reset we should use body, p, h2...